strelka_dirstream.py

```diff
@@ -133,6 +141,18 @@ def delete_file(self, path):
             logging.error(f"{self.name}: failed to delete"
                           f" file {path} (PermissionError)")
 
+    def move_file(self, path):
+        """Archives files."""
```

Comment:
Found one last reference to "archive" here -- I'll merge this in once it's switched to "moves"

Reply:
may bad! fixing now.

Reply:
fixed in dd26e3e